docs/section5/Rule5-22.md

```diff
@@ -0,0 +1,36 @@
+
+# Envelope - Rule 5-22  
+
+**Rule ID:** 5-22  
+**Rule Description:** The baseline fenestration area for an existing building shall equal the existing fenestration area prior to the proposed work.  
+**Rule Assertion:** B-RMR total (subsurface.glazed_area+subsurface.opaque_area) = expected value  
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-5(c) Building Envelope Modeling Requirements for the Baseline building  
+
+**Data Lookup:** None  
+**Evaluation Context:**  Each Data Element  
+
+**Applicability Checks:** 
+
+1. The baseline building has existing or altered spaces
+
+**Manual Checks:** Yes  
+**Function Call:**  None  
+
+## Rule Logic:
+
+- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+
+  - For each space in zone: `for space_b in zone_b.spaces:`
+
+    - Check if space is existing or altered, set rule applicability check to True: `if ( space_b.status_type == "EXISTING" ) OR ( space_b.status_type == "ALTERATION" ): rule_applicability_check = TRUE`
+
+      - Add to total number of existing or altered spaces in zone: `num_space_existing_altered += 1`
+
+**Rule Assertion:**
+
+- For each zone, if any space in zone is existing or altered: `if num_space_existing_altered > 0: CAUTION and raise_warning "PART OR ALL OF ZONE IS EXISTING. THE BASELINE VERTICAL FENESTRATION AREA FOR EXISTING ZONES MUST EQUAL TO THE FENESTRATION AREA PRIOR TO THE PROPOSED SCOPE OF WORK. THE BASELINE FENESTRATION AREA IN ZONE MUST BE CHECKED MANUALLY."`
+
```

Comment:
1. We agreed to changed CAUTION to UNDETERMINED, but this perhaps should be addressed in the code and not here.
2. This calls for showing CAUTION for each affected zone. I think instead we want UNDERTERMINED outcome if at least one zone is existing, and then list the affected zones in the accompanying (optional) message

Reply:
Updated.